### PR TITLE
Load icons using @svgr

### PR DESCRIPTION
**Pull request by kerryarchibald**
Changes in element-web: https://github.com/vector-im/element-web/pull/21248

Current icon situation is a mixture of two approaches -
- using svgs in css as `mask-image`, can control colour, expensive for css evaluation, fussy, heaps of duplication of styles
- loading svgs as images, can't control colour

This PR extends the .svg modules types and adds a POC in `QuickSettingsButton`
Loader changes in element-web https://github.com/vector-im/element-web/pull/21248
@svgr automatically applies `role="img"` and `aria-hidden` to generated svg components.

Further improvements:
- groom svgs to set `fill: currentColor` so most of the time icons will not need any particular styling

---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

Preview: https://pr7928--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by t3chguy** [comment]
What will be a11y impact of this be? Presumably now there'll be an img node without an alt text? Given that we use mask-images for presentational content we should mark is as such with aria.

**Comment by kerryarchibald** [comment]
@t3chguy the current approach has img tags without alt text, you can unignore `jsx-a11y/alt-text` in eslintrc to see these.

I've applied `aria-hidden` to each of the icons in this PR by hand, but it's possible in the svgr config to automatically apply attributes.

**Comment by t3chguy** [comment]
> I've applied aria-hidden to each of the icons in this PR by hand, but it's possible in the svgr config to automatically apply attributes.

Not sure whether that or role=presentation is more semantically accurate. It'd be great if SVGR could set the right one by default but let us unset it on a case-by-case basis as sometimes the image carries semantic context

File: src/components/views/spaces/QuickSettingsButton.tsx [hunk 1]
```diff
@@ -59,20 +63,26 @@ const QuickSettingsButton = ({ isPanelCollapsed = false }) => {
                 { _t("All settings") }
             </AccessibleButton>
 
-            <h4 className="mx_QuickSettingsButton_pinToSidebarHeading">{ _t("Pin to sidebar") }</h4>
+            <h4 className="mx_QuickSettingsButton_pinToSidebarHeading">
+                <PinUprightIcon aria-hidden className="mx_QuickSettingsButton_icon" />
```
Comment (t3chguy) [inline src/components/views/spaces/QuickSettingsButton.tsx:67]: we don't need to specify aria-hidden on each anymore

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

**Pull request merged**